### ISSUE TITLE
Crash when playback starts or stops 16.5.3

### ISSUE THREAD

**Issue by SPI0** [koying/SPMC#601]
Hi koying, thanks for this superb fork.

Unfortunately i have issues when i playback a file and spmc crashes with refresh rate switching on the newer 16.5.x. Reminds me of the crash which occurred with the older 16.4.x versions.

Here's a paste of a crash which occured after stopping video playback: http://paste.semperpax.com/pgqnadzkr

Thanks for your help!

**Comment by nekromantik-zz** [comment]
getting the same. no version of 16.5.x has fixed this so im back to 16.4

**Comment by koying** [comment]
It crashes in some script/plugin, for both of you afaict
No idea why, and surely no idea why it worked in 16.4 and stopped working in 16.5

**Comment by SPI0** [comment]
Thanks, i'll try a fresh install.

**Comment by koying** [comment]
Random idea...
Could you test http://download.semperpax.com/spmc/android-arm/SPMC-16.5.3-spmc-be672c0-armeabi-v7a_gplay.apk

**Comment by SPI0** [comment]
Yes it's still crashing.

**Comment by koying** [comment]
Another test: http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-f0fbe04-armeabi-v7a.apk
(This one has a different package name)

**Comment by SPI0** [comment]
Still crashing :(  http://paste.semperpax.com/pacunfbf7
Thanks for the different name better to test ;)

**Comment by koying** [comment]
Another try: http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-8c9f00d-armeabi-v7a.apk
BTW, what are you using as a remote?

**Comment by SPI0** [comment]
The standard shield remote.
Still crashing: http://paste.semperpax.com/p4ti2twnf

**Comment by koying** [comment]
Can you use something else, like the controller, just for test?

**Comment by SPI0** [comment]
Used only the controller, thought you are on to sth because it took some trys to crash it ;)
Log: http://paste.semperpax.com/pjl4nymga

**Comment by nekromantik-zz** [comment]
I use my TV remote to control mine, Never use controller.

**Comment by koying** [comment]
@nekromantik Could you repost a log, please. Maybe not the same issue

**Comment by koying** [comment]
Next try: http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-cc3027a-armeabi-v7a.apk
I have some hope ;)

**Comment by SPI0** [comment]
I hope too, thanks for fixing ;)
Unfortunately crashed again: http://paste.semperpax.com/phskgkcik

**Comment by koying** [comment]
Actually, there's some progress. It's another one, probably the same as @nekromantik

**Comment by koying** [comment]
Could you redo a test with debugging enabled, please

**Comment by SPI0** [comment]
Here you go: http://paste.semperpax.com/pjy3bj4oo

**Comment by nekromantik-zz** [comment]
sorry wont be able to test anything until after new year now

**Comment by koying** [comment]
Another one: http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-ca693de-armeabi-v7a.apk

**Comment by rainman74** [comment]
http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-ca693de-armeabi-v7a.apk

...crashes on Galaxy Tab S2 immediately at app start - crash handler isn't triggered and no log is written :-(

logcat: https://1drv.ms/t/s!Au4QgT0yuh6DjSvqNBslLyya008k

**Comment by SPI0** [comment]
Nope :( http://spmclog.semperpax.com/pxfmurvut

**Comment by koying** [comment]
@rainman74 :
12-23 01:57:48.176 12965 12979 D SPMC_STD: Could not init logging classes. Permission errors on ~/.spmc (/storage/1576-214E/Android/data/com.semperpax.spmc16/files/.spmc/
12-23 01:57:48.176 12965 12979 D SPMC_STD: temp/)

No access to home dir, for whatever reason. Please create a separate issue

**Comment by koying** [comment]
@SPI0 Out of ideas for the latest one.
Please redo a test to see if it's not a transient one. Thanks

**Comment by SPI0** [comment]
Had to copy log. Handler didn't trigger: http://paste.semperpax.com/puavsyd19

**Comment by koying** [comment]
logcat, maybe?

**Comment by SPI0** [comment]
Handler triggered. Dunno how to get the logcat ^^ 
http://paste.semperpax.com/pqy7kdk3l

**Comment by rainman74** [comment]
@koying http://download.semperpax.com/spmc/android-arm/SPMC-16.5.4-spmc-ca693de-armeabi-v7a.apk runs fine now on Tab S2.

The access Problem had something to do with changed folder name of test build!

**Comment by nekromantik-zz** [comment]
this one seems to have solved the issue for me at least.
http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-ca693de-armeabi-v7a.apk

No more crashes after playing or stopping a movie.
Only tried it 2-3 times though. 
Thanks! 

As its a test version it installed along side my existing SPMC install rather then updating old one.

Merry Xmas and happy new year koying!

**Comment by koying** [comment]
@nekromantik All those fixes are in 16.5.4

**Comment by nekromantik-zz** [comment]
@koying the version of 16.5.4 on playstore I still get crashes with when playing and stopping media.

**Comment by koying** [comment]
Doesn't make much sense, it's actually exactly the same code ;)

**Comment by nekromantik-zz** [comment]
@koying hmm yeah that does not make sense.

**Comment by SPI0** [comment]
@koying was my latest logcat helpful?

**Comment by 99rook99** [comment]
Playstore has 16.5.3,  at least on my system

**Comment by nekromantik-zz** [comment]
spoke too soon it crashed on media stop.
:(

**Comment by nekromantik-zz** [comment]
another log
pcm77hays

same error?

**Comment by koying** [comment]
Yep. This *should* definitely get rid of it: http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-972797d-armeabi-v7a.apk

**Comment by SPI0** [comment]
Thanks 👍 Seems not to crash anymore :)

**Comment by nekromantik-zz** [comment]
crashes seem to have died down.
if I get any more on the new version I will post here.
thanks

**Comment by i286** [comment]
Can I also chime in that your latest http://download.semperpax.com/spmc/android-arm/SPMC-16.5.x-972797d-armeabi-v7a.apk has fixed the problem for me too. I was getting crashes on stopping local network media that I don't remember seeing before 16.5.3.

Cheers!

**Comment by blinky2134** [comment]
Hi,

I'm a non programmer so go easy on me!  I am having a problem on spmc, often when I stop a film or hit the back button on my remote the screen goes black. If I press buttons on the remote I hear the sound but I need to go to the android home screen, clean the memory then go back into Kodi. Would this be resolved by downloading the apk above? Is this apk also the one on the play store so I can just do an update from the play store?
